jsonnet/thanos-querier.jsonnet

```diff
@@ -29,9 +29,7 @@ local authorizationRole =
   local config = super._config,
 
   thanos+:: {
-    variables+:: {
-      image: config.imageRepos.openshiftThanos + ':' + config.versions.openshiftThanos,
-    },
+    image:: config.imageRepos.openshiftThanos + ':' + config.versions.openshiftThanos,
```

Comment:
I had to update this for the generation to work, I assume this was due to some change in kube-thanos cc @metalmatze ?

Reply:
yes I can confirm: https://github.com/thanos-io/kube-thanos/commit/aae64db90a24d1445f373f94b5d298c141ef3d41#diff-2287ff014e43baab107e2583776d124b

Thanks!

Reply:
Yup. Exactly right. :grin: :innocent: